client/asset/dcr/dcr.go

```diff
@@ -1586,19 +1829,39 @@ func (dcr *ExchangeWallet) FundOrder(ord *asset.Order) (asset.Coins, []dex.Bytes
 		dcr.log.Errorf("calcBumpRate error: %v", err)
 	}
 
-	coins, redeemScripts, sum, inputsSize, err := dcr.fund(orderEnough(ord.Value, ord.MaxSwapCount, bumpedMaxRate))
-	if err != nil {
-		return nil, nil, fmt.Errorf("error funding order value of %s DCR: %w",
-			amount(ord.Value), err)
-	}
-
+	// If a split is not requested, but is forced, create an extra output from
+	// the split tx to help avoid a forced split in subsequent orders.
+	var extraSplitOutput uint64
 	useSplit := cfg.useSplitTx
 	if customCfg.Split != nil {
 		useSplit = *customCfg.Split
 	}
 
-	// Send a split, if preferred.
-	if useSplit && !ord.Immediate {
+	changeForReserves := useSplit && cfg.unmixedAccount == ""
```

Comment:
Just the non-change outputs.  The change output goes into the unmixed account, according to signTxAndAddChange+depositAccount.
Mixing epochs are unfortunately kinda long so I don't think we can consider such change as available

Reply:
I see now.. but why does that change need to go into the unmixed account when we are doing a split? Couldn't it go right back into the trading account?

Reply:
It's debatable, but all the input have been linked with each other.  They're also now linked with a dex order.

Reply:
Wouldn't the second output created for the reserves have the same issue then?

Reply:
Yeah, and same as when an order that's funded by a split output is cancelled with no fill, that output stays in trading account.

But if we move nothing into unmixed, a wallet will quickly end up with nothing in the mixed account, I think.  Meaning the entire previously mixed set gets linked.  It would be down to deposits, redeems, and change on matched orders.  And the last category would not exist with splits in use.